atlp-rwanda/rca-phantom-team2-fe
# Setup CI & QA pipeline

#### What does this PR do?

- Implements add a todo task

#### Description of Task to be completed?

- User can be able to add a todo task

#### How should this be manually tested?

- Clone the repo

- checkout to branch `ch-setup-ci-and-qa-pipeline`

- Run `npm install` to install modules

- Run `npm run test` to check if the tests pass

- Check out the ReadME.md file to see the added badges

#### What are the relevant Trello stories?

- Link to [Trello](https://trello.com/c/VanyBcCq/27-setup-continuous-integration-quality-assurance-pipeline) implemented story
